lib/datasource/terraform-provider/index.ts

```diff
@@ -63,6 +64,18 @@ export class TerraformProviderDatasource extends TerraformDatasource {
     return dep;
   }
 
+  override handleSpecificErrors(err: HttpError): void {
```

Comment:
Needs a test. 😏

Reply:
`TerraformProviderDatasource` and `TerraformModuleDatasource` share the same `handleSpecificErrors`. You could move this to `TerraformDatasource`.

Reply:
Good idea 👍